File: GZCTF/Models/Request/Account/LoginModel.cs
using System.ComponentModel.DataAnnotations;

namespace CTFServer.Models.Request.Account;

/// <summary>
/// 登录
/// </summary>
public class LoginModel
{
    /// <summary>
    /// 用户名或邮箱
    /// </summary>
    [Required]
    public string? UserName { get; set; }

    /// <summary>
    /// 密码
    /// </summary>
    [Required]
    [MinLength(6, ErrorMessage = "密码过短")]
    public string? Password { get; set; }

    /// <summary>
    /// Google Recaptcha Token
    /// </summary>
    public string? GToken { get; set; }
}

File: GZCTF/ClientApp/src/client/models/LoginModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

/**
 * 登录
 */
export type LoginModel = {
    /**
     * 用户名或邮箱
     */
    userName: string;
    /**
     * 密码
     */
    password: string;
    /**
     * Google Recaptcha Token
     */
    gToken?: string | null;
};
